src/NTS.Warp/ACL/Factories/AthleteFactory.cs

```diff
@@ -3,21 +3,20 @@
 using NTS.Warp.ACL.Entities.Countries;
 using NTS.Warp.ACL.Enums;
 using NTS.Warp.ACL.Models;
-using NTS.Warp.Features.Judge.Models;
 
 namespace NTS.Warp.ACL.Factories;
```

Comment:
Changes in the `ACL` namespace take into account that we aim to delete it entirely when we integrate NTS, right?